test_runner/fixtures/neon_fixtures.py

```diff
@@ -453,6 +453,7 @@ def __init__(
         pageserver_get_vectored_concurrent_io: str | None = None,
         pageserver_tracing_config: PageserverTracingConfig | None = None,
         pageserver_import_config: PageserverImportConfig | None = None,
+        disable_kick_secondary_downloads: bool = False,
```

Comment:
`| None = None` is the best - don't add any values (false/true) if it's not specified.
Otherwise compatibility tests will fail